packages/actions/scoreboard/src/components/Scoreboard.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import {useEffect} from 'react'
+import {FormattedMessage} from 'react-intl'
+import styled from 'styled-components'
+import {LoadingSpinner, Typography} from 'tocco-ui'
+
+const StyledContainer = styled.div`
+  width: 148px;
+  height: 148px;
+  position: relative;
+  align-self: center;
+`
+
+const StyledContent = styled.div`
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`
+const Scoreboard = ({data, fetchData}) => {
+  useEffect(() => {
+    fetchData()
+  }, [fetchData])
+
+  let content
+
+  if (data === undefined) {
+    content = <LoadingSpinner />
+  } else if (data === null) {
+    content = (
+      <Typography.P>
+        <FormattedMessage id="client.scoreboard.fetchFailed" />
+      </Typography.P>
+    )
+  } else {
+    content = data.map(entry => {
+      const defaultDisplay = <p>{entry.participantBean.defaultDisplay}</p>
+      const rank = <p>Ranking: {entry.ranking}</p>
+      const points = <p>Points: {entry.points}</p>
+
+      const returnValue = [defaultDisplay, rank, points]
+      return returnValue
+    })
+  }
+  return (
+    <StyledContainer>
+      <StyledContent>{content}</StyledContent>
+    </StyledContainer>
+  )
+}
+
+Scoreboard.propTypes = {
+  data: PropTypes.arrayOf(
```

Comment:
suggestion(blocking): I think this is not 100% correct.

Now it would be something like this:
```
data = [{key: 1, entity: '', defaultDisplay: ''}, {points: 1, ranking: 1}]
```
This means: an _Array_ of two separate _Objects_. First Object has three properties (_key_, _entity_, _defaultDisplay_) and second Object has two properties (_points_, _ranking_)

but it should be:

```
data = [{participantBean: {key: 1, entity: '', defaultDisplay: ''}, points: 1, ranking: 1}]
```
so you have an _Array_ of _Objects_. Those Objects have three properties (_participantBean_, _points_, _ranking_) and _participantBean_ is an Object with three properties (_key_, _entity_, _defaultDisplay_)